src/components/ListItem/ListItem.tsx

```diff
@@ -56,8 +56,6 @@ const useStyles = (isRecent: boolean) =>
         },
         '&>$icon': {
           visibility: 'inherit',
-          display: 'flex',
-          alignItems: 'center'
```

Comment:
This is applied on `'&:focus:not(:hover), &.selected':` state of `ListItem` move it below make the return icon always visible no?